swan-cern/values.yaml

```diff
@@ -268,29 +268,30 @@ fluentd:
           path: ca-bundle.crt
 
 gpu-operator:
-  enabled: true
+  enabled: false
   validator:
     repository: registry.cern.ch/kubernetes
-    version: v22.9.1
+    version: v24.6.2
   operator:
     repository: registry.cern.ch/kubernetes
-    version: v22.9.1
+    version: v24.6.2
     defaultRuntime: containerd
     initContainer:
       repository: registry.cern.ch/kubernetes
       version: 11.4.2-base-ubi8
   driver:
     repository: registry.cern.ch/kubernetes
     image: nvidia-gpu-driver
-    version: "v550.54.15-6.6.13-200.fc39.x86_64"
+    version: "v550.54.15-6.8.4-200.fc39.x86_64"
```

Comment:
Can we have this as a separate commit, since it is not related to the gpu-operator update?